saleor/webhook/transport/synchronous/circuit_breaker/storage.py

```diff
@@ -118,3 +131,11 @@ def register_event_returning_count(
         except (RedisError, IndexError):
             logger.warning(self.WARNING_MESSAGE, exc_info=True)
             return 0
+
+    def close_breaker(self, app_id: int):
```

Comment:
Storage classes themselves are not really concerned with the idea of breakers, they don't even have to be used in the context of circuit breakers. Therefore maybe the naming relating to clearing the state for a given app would be more fitting.

Reply:
Changed to `clear_state_for_app`.